tools/topology/sof-hda-generic-idisp.m4

```diff
@@ -90,3 +119,15 @@ PCM_PLAYBACK_ADD(HDMI3, 3, PIPELINE_PCM_4)
 DAI_CONFIG(HDA, 0, 1, iDisp1)
 DAI_CONFIG(HDA, 1, 2, iDisp2)
 DAI_CONFIG(HDA, 2, 3, iDisp3)
+DAI_CONFIG(DMIC, 0, 6, dmic32,
+           DMIC_CONFIG(1, 500000, 4800000, 40, 60, 48000,
+                DMIC_WORD_LENGTH(s32le), DMIC, 0,
+                PDM_CONFIG(DMIC, 0, STEREO_PDM0)))
```

Comment:
@RanderWang any reason to use only STEREO?

Reply:
yes,  I will consider 4ch occasion.